docker-compose.yml

```diff
@@ -1,4 +1,32 @@
 services:
+  nginx:
+    image: nginx:alpine
+    cap_add:
+      - NET_BIND_SERVICE
+    ports:
+      - "80:80"
+      - "443:443"
+      - "8443:8443"
```

Comment:
we shouldn't clutter the host's ports. Can we use only "weird" ports like 8443 and 2096 maybe?

Also please add a reference to https://developers.cloudflare.com/fundamentals/reference/network-ports/ so we know why we have this limitation

Reply:
These are the default ports for HTTP (port 80) and HTTPS (port 443) web servers. Web browsers use these ports by default.